hi-ml/src/health_ml/run_ml.py

```diff
@@ -244,6 +243,21 @@ def after_ddp_cleanup(self, old_environ: Dict) -> None:
         if torch.distributed.is_initialized():  # type: ignore
             torch.distributed.destroy_process_group()  # type: ignore
 
+    def end_training(self, old_environ: Dict) -> None:
```

Comment:
Consider renaming `old_environ` to something more telling, like `environ_before_training`?

Reply:
done